### ISSUE TITLE
PHPUnit problem on Debian 7

### ISSUE THREAD

**Issue by AntonioCS** [jippi/puppet-php#104]
I had this on my manifest file:
```puppet
Php::Extension <| |>
  # Configure extensions
  -> Php::Config <| |>
  # Reload webserver
  ~> Service["apache2"]

class { 
	[
	'php::apache','php::cli','php::extension::curl','php::extension::imagick','php::extension::mcrypt',
	'php::extension::memcached','php::extension::mysql','php::extension::xdebug',	
	'php::composer','php::composer::auto_update'
	]:
}

class { 'php::phpunit':
	ensure => latest,
}
```
Once I did vagrant provision I got this:

==> machine_debian_dev: Error: Could not find resource 'Exec[php::pear::auto_discover]' for relationship on 'Package[pear.phpunit.de/PHPUnit]' on node machine.dev
==> machine_debian_dev: Error: Could not find resource 'Exec[php::pear::auto_discover]' for relationship on 'Package[pear.phpunit.de/PHPUnit]' on node machine.dev

It all worked (except that xdebug installed to the wrong place) after I removed the php::phpunit clas

These are the module I have currently installed

├── nodes-php (v0.9.0)
├── puppetlabs-apache (v1.2.0)
├── puppetlabs-apt (v1.7.0)
├── puppetlabs-concat (v1.1.2)
├── puppetlabs-git (v0.3.0)
├── puppetlabs-inifile (v1.2.0)
├── puppetlabs-stdlib (v4.4.0)
└── puppetlabs-vcsrepo (v1.2.0)

I assume there is a dependency on PEAR module.

My box is debian 7 and got the image from https://vagrantcloud.com/janschumann/boxes/debian-7

**Comment by tizzo** [comment]
This issue should be closeable now that PEAR is [no longer a supported way to install phpunit](https://github.com/sebastianbergmann/phpunit/wiki/End-of-Life-for-PEAR-Installation-Method) - in fact this class should probably be removed from the module.

**Comment by schkovich** [comment]
@igalic Isn't this fixed by #128 and #136? :)

**Comment by igalic** [comment]
given the lack of recent feedback, i'd say we close it ;)
